DigitalLearningSolutions.Data/Services/CourseService.cs

```diff
@@ -334,17 +330,17 @@ public IEnumerable<string> GetTopicsForCentreAndCentrallyManagedCourses(int cent
             return courseTopicsDataService.GetCourseTopicsAvailableAtCentre(centreId).Select(c => c.CourseTopic);
         }
 
-        public (
-            IEnumerable<CourseStatisticsWithAdminFieldResponseCounts> courses,
-            IEnumerable<string> categories,
-            IEnumerable<string> topics
-            ) GetCentreCourseCategoriesTopics(int centreId, int? categoryId)
+        public CentreCourseDetails GetCentreCourseDetails(int centreId, int? categoryId)
         {
-            return (
+            var details = new CentreCourseDetails();
+
+            (details.Courses, details.Categories, details.Topics) = (

```

Comment:
Minor, but might as well make a constructor to assign these.

Reply:
I agree, we use nullability a fair bit on this project, so it's good if we can ensure that properties are always initialized if they're not marked as nullable.